examples/movement/physics_in_fixed_timestep.rs

```diff
@@ -0,0 +1,191 @@
+            transform: Transform::from_scale(Vec3::splat(0.3)),
+            ..default()
+        },
+        Velocity::default(),
+        PhysicalTranslation::default(),
+    ));
+}
+
+/// Spawn a bit of UI text to explain how to move the player.
+fn spawn_text(mut commands: Commands) {
+    commands
+        .spawn(NodeBundle {
+            style: Style {
+                position_type: PositionType::Absolute,
+                bottom: Val::Px(12.0),
+                left: Val::Px(12.0),
+                ..default()
+            },
+            ..default()
+        })
+        .with_children(|parent| {
+            parent.spawn(TextBundle::from_section(
+                "Move the player with WASD",
+                TextStyle {
+                    font_size: 25.0,
+                    ..default()
+                },
+            ));
+        });
+}
+
+/// Handle keyboard input to move the player.
+fn handle_input(keyboard_input: Res<ButtonInput<KeyCode>>, mut query: Query<&mut Velocity>) {
+    /// Since Bevy's default 2D camera setup is scaled such that
+    /// one unit is one pixel, you can think of this as
+    /// "How many pixels per second should the player move?"
+    const SPEED: f32 = 210.0;
+    for mut velocity in query.iter_mut() {
+        velocity.0 = Vec3::ZERO;
+
+        if keyboard_input.pressed(KeyCode::KeyW) {
+            velocity.y += 1.0;
+        }
+        if keyboard_input.pressed(KeyCode::KeyS) {
+            velocity.y -= 1.0;
+        }
+        if keyboard_input.pressed(KeyCode::KeyA) {
+            velocity.x -= 1.0;
+        }
+        if keyboard_input.pressed(KeyCode::KeyD) {
+            velocity.x += 1.0;
+        }
+
+        // Need to normalize and scale because otherwise
+        // diagonal movement would be faster than horizontal or vertical movement.
+        velocity.0 = velocity.normalize_or_zero() * SPEED;
+    }
+}
+
+/// Advance the physics simulation by one fixed timestep. This may run zero or multiple times per frame.
+///
+/// Note that since this runs in `FixedUpdate`, `Res<Time>` would be `Res<Time<Fixed>>` automatically.
+/// We are being explicit here for clarity.
+fn advance_physics(
+    fixed_time: Res<Time<Fixed>>,
+    mut query: Query<(&mut PhysicalTranslation, &Velocity)>,
+) {
+    for (mut physical_translation, velocity) in query.iter_mut() {
+        physical_translation.0 += velocity.0 * fixed_time.delta_seconds();
+    }
+}
+
+fn update_rendered_transform(
+    fixed_time: Res<Time<Fixed>>,
+    mut query: Query<(&mut Transform, &PhysicalTranslation)>,
+) {
+    for (mut transform, physical_translation) in query.iter_mut() {
+        let last_rendered_translation = transform.translation;
+        // The overstep fraction is a value between 0 and 1 that tells us how far we are between two fixed timesteps.
+        let alpha = fixed_time.overstep_fraction();
+
+        let next_rendered_translation =
+            last_rendered_translation.lerp(physical_translation.0, alpha);
```

Comment:
I'd be happy if someone could verify that this is the way to go. I merely adapted it from @cBournhonesque's [relvated chapter in the Lightyear Book](https://cbournhonesque.github.io/lightyear/book/concepts/advanced_replication/visual_interpolation.html)

Reply:
Update: I got it wrong, this should interpolate between the last `PhysicalTransform`, not `Transform`.